app/components/EulaModal.js

```diff
@@ -26,6 +26,15 @@ export const EulaModal = ({ selectedLocale, continueFunction }) => {
   // Pull the EULA in the correct language, with en as fallback
   const eulaPath = EULA_FILES[selectedLocale] || en;
 
+  // Any links inside the EULA should launch a separate browser otherwise you can get stuck inside the app
+  shouldStartLoadWithRequestHandler = (webViewState) => {
+    if (webViewState.url != 'about:blank') {
```

Comment:
Could you pull `about:blank` out into a variable with a name along the lines of `const DEFAULT_EULA_URL = 'about:blank'` ? I had no idea that was what that string was until you explained to me so probably would be useful to other folks as well.

Also in general - best practice to do `!==` over `!=` just to be sure we're [forcing strict equality comparison.](https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a)

Reply:
One other note - I would put this at the module level, e.g. 

```js
const DEFAULT_EULA_URL = 'about:blank'
export const EulaModal = ({ selectedLocale, continueFunction }) => {
```